parser.y

```diff
@@ -1824,6 +1868,8 @@ ColumnOption:
 		// See https://dev.mysql.com/doc/refman/5.7/en/create-table.html
```

Comment:
we should implement `EnforcedOrNotOpt` for `"CHECK" '(' Expression ')'` item of `ColumnOption`,too.
but if I append `EnforcedOrNotOpt` to `"CHECK" '(' Expression ')'`, `goyacc` will find conflict around `NOT`,I have no idea how to fix it, can you help me? @kennytm

Reply:
@leoppro What does it conflict with and is this a shift/reduce or reduce/reduce conflict? Can you assign a precedence an arm?